README.rst

```diff
@@ -187,6 +187,12 @@ To enable New Relic monitoring for an environment:
   include any New Relic configuration, so it's safe to put other settings
   in ``project.sls`` even if you're not using New Relic in every environment.
 
+* If you are using elasticsearch and would like New Relic monitoring of that as well,
+  add to pillar somewhere::
+
+  # project.sls or <environment>.sls
+  elasticsearch_newrelic: true
```

Comment:
I think these lines might need to be indented a few more spaces to get proper formatting. Currently it looks like this:
![image](https://cloud.githubusercontent.com/assets/4413/9960679/00689816-5dea-11e5-84e9-12d0ea210991.png)